lib/bike_brigade_web/live/components/location_form.ex

```diff
@@ -1,63 +1,142 @@
 defmodule BikeBrigadeWeb.Components.LocationForm do
```

Comment:
We should back this file out to what it used to be so we dont get rid of the old form